examples/direct_fidelity_estimation.py

```diff
@@ -185,17 +199,48 @@ def _estimate_pauli_traces_general(qubits: List[cirq.Qid],
         cirq.DensityMatrixTrialResult,
         dense_simulator.simulate(circuit)).final_density_matrix
 
-    pauli_traces = []
+    pauli_traces: List[PauliTrace] = []
     for P_i in itertools.product([cirq.I, cirq.X, cirq.Y, cirq.Z],
                                  repeat=n_qubits):
         pauli_string = cirq.PauliString(dict(zip(qubits, P_i)))
         rho_i, Pr_i = compute_characteristic_function(circuit, pauli_string,
                                                       qubits,
                                                       clean_density_matrix)
-        pauli_traces.append({'P_i': pauli_string, 'rho_i': rho_i, 'Pr_i': Pr_i})
+        pauli_traces.append(PauliTrace(P_i=pauli_string, rho_i=rho_i,
+                                       Pr_i=Pr_i))
     return pauli_traces
 
 
+@dataclass
+class TrialResult:
+    """
+    Contains the results of a trial, either by simulator or actual run
+    """
+    # The index in the list of Pauli traces.
+    i: int
+    # The measured state for that trial under the Pauli state indexed by i
```

Comment:
This is not a measured state. See comment above. Whatever you decide there, please make this comment consistent with the one above.

Reply:
I changed it, but I wasn't as sure. Does this look reasonable?